### PR TITLE
proofread-loic

### PR DESCRIPTION
**Pull request by dmerejkowsky**

### CODE REVIEW

File: content/post/0071-giving-mypy-a-go.md [hunk 1]
```diff
@@ -98,16 +83,28 @@ There are many things said about types annotations like:
 
 Those may be true, but they are hard to measure.
 
-So I ask myself something something else:
+So I ask myself something else:
```
Comment (banetl) [inline content/post/0071-giving-mypy-a-go.md]: -> asked myself  (if you wanna keep the following sentences in past tense)

Comment (dmerejkowsky) [reply content/post/0071-giving-mypy-a-go.md]: oups

File: content/post/0071-giving-mypy-a-go.md [hunk 2]
```diff
@@ -71,24 +71,9 @@ This is called "gradual typing".
...
+## Some definitions
 
-tsrc is the tool we use at [my work](https://tanker.io) to manage multiple git repositories and automate GitHub and GitLab review automation.[^1]
-
-I chose it because:
-
-* It's not too big, so the experiment will not take too long.
-* It's not too small, so we'll have enough experimental data.
-
-I also chose it because a lot of tools were already used in the hope of catching bugs:
```
Comment (banetl) [inline content/post/0071-giving-mypy-a-go.md]: This list was quite nice actually

Comment (dmerejkowsky) [reply content/post/0071-giving-mypy-a-go.md]: yeah i moved it below

Comment (banetl) [reply content/post/0071-giving-mypy-a-go.md]: Showed that the previous stack not fail proof and the use of mypy could still improve the code further

File: content/post/0071-giving-mypy-a-go.md [hunk 3]
```diff
@@ -234,13 +231,21 @@ Here, mypy forced us to follow an implicit convention. There are two ways to rep
 
 An other win for mypy.
 
-### Bugs found
+## Bugs found
+
+We already use a lot of tools in the hope of catching bugs:
```
Comment (banetl) [inline content/post/0071-giving-mypy-a-go.md]: Aw nice didn't see you put it here. Much better than above

File: README.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
-Sources of https://dmerej.info/blog, built with [hugo](https://gohugo.io/)
+# Description
+
+This repository contain the sources of https://dmerej.info/blog, built with [hugo](https://gohugo.io/)
```
Comment (banetl) [inline README.md]: contains

**Pull request merged**